INSTALL.md

```diff
@@ -73,10 +79,14 @@ filter-repo only consists of a few files that need to be installed:
 So, installation might look something like
 ```
 cp -a git-filter-repo $(git --exec-path)
-cp -a git-filter-repo.1 $(git --man-path)/man1
-cp -a git-filter-repo.html $(git --html-path)
 ln -s $(git --exec-path)/git-filter-repo \
     $(python -c "import site; print(site.getsitepackages()[-1])")/git_filter_repo.py
+
+# If you've cloned the repo directly you need to make the doc files
```

Comment:
Perhaps change this to "If you've cloned the repo directly, grab the documentation files via".